api/controllers/swagger.go

```diff
@@ -171,6 +187,36 @@ func (s *SwaggerController) ServeSwaggerSpec(c *fireball.Context) (fireball.Resp
 					},
 				},
 			},
+			"/environment/{id}/link": map[string]swagger.Method{
+				"post": {
+					Summary: "Create an Environment Link",
+					Tags:    []string{"Environment"},
+					Parameters: []swagger.Parameter{
+						swagger.NewStringPathParam("id", "ID of the source environment", true),
```

Comment:
[nit] should we keep `source_id` here to map to `source_id` in delete?

Reply:
✅